packages/atlas/src/hooks/useVideosInCategories.ts

```diff
@@ -0,0 +1,14 @@
+import { useCategories } from '@/api/hooks/categories'
+import { VideoCategoryData } from '@/config/categories'
+
+export const useVideosInCategories = () => {
+  const { categories } = useCategories()
```

Comment:
Can we rename this hook (and maybe the associated file) to `useQnCategories` so it's clear this isn't about display categories?